Makefile

```diff
@@ -93,12 +106,47 @@ format: ## format Jsonnet sources and libraries
 	bin/jsonnetfmt -i $(sources) $(libraries)
 .PHONY: format
 
-lint: ## lint Jsonnet sources and libraries
+lint-black: ## lint back-end python sources with black
+	@echo 'lint:black started…'
+	@$(COMPOSE_RUN_ACL) black acl tests
+.PHONY: lint-black
+
+lint-flake8: ## lint back-end python sources with flake8
+	@echo 'lint:flake8 started…'
+	@$(COMPOSE_RUN_ACL) flake8
+.PHONY: lint-flake8
+
+lint-isort: ## automatically re-arrange python imports in back-end code base
+	@echo 'lint:isort started…'
+	@$(COMPOSE_RUN_ACL) isort --atomic .
+.PHONY: lint-isort
+
+lint-pylint: ## lint back-end python sources with pylint
+	@echo 'lint:pylint started…'
+	@$(COMPOSE_RUN_ACL) pylint acl tests
+.PHONY: lint-pylint
+
+lint-bandit: ## lint back-end python sources with bandit
+	@echo 'lint:bandit started…'
+	@$(COMPOSE_RUN_ACL) bandit -qr acl
+.PHONY: lint-bandit
```

Comment:
Even if there is also one script in Python for the fixtures, maybe could we apply linters for the whole project? WDYT?

Reply:
Yeah sure! Good catch, thx!